src/lib.rs

```diff
@@ -1,11 +1,16 @@
 //! The Interactive-clap library is an add-on for the Command Line Argument
-//! Parser (https://crates.io/crates/clap). Interactive-clap allows you to parse
+//! Parser <https://crates.io/crates/clap>. Interactive-clap allows you to parse
```

Comment:
```suggestion
//! Parser ([`CLAP`](https://crates.io/crates/clap>)). Interactive-clap allows you to parse
```

I think, it's better like that.